job-server/config/dse.conf

```diff
@@ -14,12 +14,12 @@ spark {
 
   jobserver {
     port = 8090
-    jar-store-rootdir = /tmp/jobserver/jars
+    jar-store-rootdir = ${HOME}/.spark-jobserver/jars
 
     jobdao = spark.jobserver.io.JobFileDAO
 
     filedao {
-      rootdir = /tmp/spark-job-server/filedao/data
+      rootdir = ${HOME}/.spark-jobserver/filedao/data
```

Comment:
This change will cause "historical data" loss after upgrade. Jars, info about past jobs will be missing after upgrade. Do we want to do it in patch release?

Reply:
I think that people can stick to old configuration. I don't know if configuration files are being changed when upgrading patch release at all (my guess would be no).
All in all, this is something we should ask @brianmhess

Reply:
On the other hand, the ticket is about not creating such directories with 777 permissions because they would not pass security audits. So I don't think we have other ways to fix that than notifying users to copy their files to respective home directories.